lib/semantics/expression.cc

```diff
@@ -459,16 +459,15 @@ MaybeExpr ExpressionAnalyzer::Analyze(const parser::RealLiteralConstant &x) {
   // letter used in an exponent part (e.g., the 'E' in "6.02214E+23")
   // should agree.  In the absence of an explicit kind parameter, any exponent
   // letter determines the kind.  Otherwise, defaults apply.
-  auto &defaults{context_.defaultKinds()};
-  int defaultKind{defaults.GetDefaultKind(TypeCategory::Real)};
+  int defaultKind{context_.GetDefaultKind(TypeCategory::Real)};
```

Comment:
I don't think that hiding `context_.defaultKinds()` improves readability apart from saving a line of code, and it might actually make it harder to find the API for its class.

Reply:
OK